### PR TITLE
Add Iraq

### PR DESCRIPTION
**Pull request by alixunderplatz**
Feel free to comment and improve this attempt for a parser and to add/update the remaining files :)

ref #1434

### CODE REVIEW

**Comment by systemcatch** [comment]
iraq

File: parsers/IQ.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+    sortedZoneKeys = '->'.join(sorted([zone_key1, zone_key2]))
+
+    r = session or requests.session()
+    response = r.get(url)
+    iraq_json = response.json()
+    iraq_exch = iraq_json['d']
+    iraq_time = iraq_json['lastmodified']
+
+    #    tie_line_mapping = {
```
Comment (corradio) [inline parsers/IQ.py]: is this still needed?

**Comment by corradio** [comment]
This is amazing guys! Looking good from here but need to merge master first.

File: parsers/IQ.py (hunk 1 above)
Comment (systemcatch) [reply parsers/IQ.py]: Not sure, @alixunderplatz ?

**Comment by systemcatch** [comment]
:+1:  I forgot to rebase before starting my additions, let me try and resolve the conflicts.

File: parsers/IQ.py (hunk 1 above)
Comment (alixunderplatz) [reply parsers/IQ.py]: If there is a way to grab the data in one step in fetch_production and then reuse it for fetch_exchange (and/or fetch_consumption), then it is unnecessary. But I don't know yet how to do this, teach me, masters! ^^

**Comment by systemcatch** [comment]
@corradio we are good to go!

**Pull request merged**